tests/test_checks.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.core.checks import run_checks
+from django.forms import ModelForm
+from django.test import TestCase, override_settings
+
+
+@override_settings(DJANGO_PROJECT_DIR=settings.BASE_DIR)
+class TestModelExcludeCheck(TestCase):
+
+    def test_form(self):
+
+        class DummyForm(ModelForm):
+
+            class Meta:
+                exclude = ("id",)
+
+        errors = run_checks()
+        errors = [e for e in errors if e.id == "maykin_django_utils.E001"]
```

Comment:
it's better call the check function directly instead of going through `django.core`, it's a plain function with a well defined expected output. Then you don't need to do any filtering

```py
from maykin_django_utils.checks import check_modelform_exclude

errors = check_modelform_exclude(None)

assert errors == [Error(...)]
```

See also https://docs.djangoproject.com/en/5.2/topics/checks/#writing-tests (and skip the integration tests)